openmc/stats/multivariate.py

```diff
@@ -779,3 +777,37 @@ def from_xml_element(cls, elem):
         """
         xyz = [float(x) for x in get_text(elem, 'parameters').split()]
         return cls(xyz)
+
+
+def spherical_uniform(r_outer, r_inner=0.0, thetas=(0., pi), phis=(0., 2*pi),
+                      origin=(0., 0., 0.)):
+    r"""
+    This function is a helper that makes it easier to create points uniformly
+    distributed in a spherical shell using the class SphericalIndependent.
+
+    It allows one to sample points independly in a spherical shell, specified
+    by (r1, r2), (theta1, theta2), (phi1, phi2) centered on the coordinates
+    (x0, y0, z0).
+
+    .. versionadded: 0.13
+
+    Parameters
+    ----------
+    radii : Iterable of float
+        Inner radius r1 and outer radius r2 of the spherical shell.
+    thetas : Iterable of float
+        Start theta1 and end theta2 of the theta-coordinates (angle relative to
+        the z-axis) in a reference frame specified by the origin parameter
+    phis : Iterable of float
+        Start phi1 and end phi2 of the phi-coordinates (azimuthal angle) in a
+        reference frame specified by the origin parameter
+    origin: Iterable of float, optional
+        coordinates (x0, y0, z0) of the center of the spherical reference frame
+        for the source. Defaults to (0.0, 0.0, 0.0)
+    """
+
+    r_dist = PowerLaw(r_inner, r_outer, 2)
+    cos_thetas_dist = Uniform(np.cos(thetas[0]), np.cos(thetas[1]))
```

Comment:
I suppose this will work but it's a little weird to me to have the second parameter be less than the first. Since cos is monotonically decreasing on (0, π), I think it's better to flip them. Also, for scalar values prefer to use `math.cos` instead of `np.cos`
```suggestion
    cos_thetas_dist = Uniform(cos(thetas[1]), cos(thetas[0]))
```
and then change the import up top to `from math import pi, cos`

Reply:
yes, havent thought about that

Reply:
This one still needs to be changed